ui/src/app/job-list/job-list.component.spec.ts

```diff
@@ -133,6 +133,8 @@ describe('JobListComponent', () => {
     tick();
 
     testComponent = fixture.debugElement.query(By.css('jm-job-list')).componentInstance;
+    testComponent.pageSize = 3;
```

Comment:
Hopefully you can revert per my other comment.

Reply:
yup, thanks